llvm/lib/Analysis/TargetLibraryInfo.cpp

```diff
@@ -337,6 +338,7 @@ static void initializeLibCalls(TargetLibraryInfoImpl &TLI, const Triple &T,
     TLI.setUnavailable(LibFunc_powl);
     TLI.setUnavailable(LibFunc_remainderl);
     TLI.setUnavailable(LibFunc_remquol);
+    TLI.setUnavailable(LibFunc_fdiml);
```

Comment:
Don't believe this is tested

Reply:
this line marks the `long double` library function unavailable for Win32
sry but i am not sure what you want to me to do here

Reply:
Add a test that shows fdiml is not recognized as a libcall on windows

Reply:
i added it because there's a comment above stating `long double` functions are not supported by Win32
are there any previous such tests that i can refer to?

Reply:
There seem to be 3 ways it's done. The most direct way has an example in TargetLibraryInfoTest.cpp in TLITestAarch64. 

There's also an underutilized looking direct test in llvm/test/tools/llvm-tli-checker, which currently only covers ps4.

The indirect way most TLI tests do is to add a negative instcombine test for some transform, e.g. test/Transforms/InstCombine/memcmp-1.ll

Reply:
tried to add a test but its failing
mingw32 is generating `fdimf`

i feel like this was not the correct way to test